`go build ./cmd/...` fails

`go build ./cmd/...` fails with the below error
```
💰 go build ./cmd/...
cmd/capd-manager/main.go:23:2: cannot find package "github.com/chuckha/cluster-api-provider-docker/actuators" in any of:
	/usr/local/Cellar/go/1.12.6/libexec/src/github.com/chuckha/cluster-api-provider-docker/actuators (from $GOROOT)
	/Users/ashisha/go/src/github.com/chuckha/cluster-api-provider-docker/actuators (from $GOPATH)
cmd/capd-manager/main.go:24:2: cannot find package "k8s.io/client-go/kubernetes" in any of:
	/usr/local/Cellar/go/1.12.6/libexec/src/k8s.io/client-go/kubernetes (from $GOROOT)
	/Users/ashisha/go/src/k8s.io/client-go/kubernetes (from $GOPATH)
cmd/capd-manager/main.go:30:2: cannot find package "sigs.k8s.io/controller-runtime/pkg/client/config" in any of:
	/usr/local/Cellar/go/1.12.6/libexec/src/sigs.k8s.io/controller-runtime/pkg/client/config (from $GOROOT)
	/Users/ashisha/go/src/sigs.k8s.io/controller-runtime/pkg/client/config (from $GOPATH)
cmd/capd-manager/main.go:31:2: cannot find package "sigs.k8s.io/controller-runtime/pkg/manager" in any of:
	/usr/local/Cellar/go/1.12.6/libexec/src/sigs.k8s.io/controller-runtime/pkg/manager (from $GOROOT)
	/Users/ashisha/go/src/sigs.k8s.io/controller-runtime/pkg/manager (from $GOPATH)
cmd/capd-manager/main.go:32:2: cannot find package "sigs.k8s.io/controller-runtime/pkg/runtime/signals" in any of:
	/usr/local/Cellar/go/1.12.6/libexec/src/sigs.k8s.io/controller-runtime/pkg/runtime/signals (from $GOROOT)
	/Users/ashisha/go/src/sigs.k8s.io/controller-runtime/pkg/runtime/signals (from $GOPATH)
cmd/capdctl/main.go:26:2: cannot find package "github.com/chuckha/cluster-api-provider-docker/execer" in any of:
	/usr/local/Cellar/go/1.12.6/libexec/src/github.com/chuckha/cluster-api-provider-docker/execer (from $GOROOT)
	/Users/ashisha/go/src/github.com/chuckha/cluster-api-provider-docker/execer (from $GOPATH)
cmd/capdctl/main.go:27:2: cannot find package "k8s.io/apimachinery/pkg/apis/meta/v1" in any of:
	/usr/local/Cellar/go/1.12.6/libexec/src/k8s.io/apimachinery/pkg/apis/meta/v1 (from $GOROOT)
	/Users/ashisha/go/src/k8s.io/apimachinery/pkg/apis/meta/v1 (from $GOPATH)
cmd/kind-test/main.go:25:2: cannot find package "github.com/chuckha/cluster-api-provider-docker/kind/actions" in any of:
	/usr/local/Cellar/go/1.12.6/libexec/src/github.com/chuckha/cluster-api-provider-docker/kind/actions (from $GOROOT)
	/Users/ashisha/go/src/github.com/chuckha/cluster-api-provider-docker/kind/actions (from $GOPATH)
cmd/kind-test/main.go:26:2: cannot find package "sigs.k8s.io/kind/pkg/cluster/constants" in any of:
	/usr/local/Cellar/go/1.12.6/libexec/src/sigs.k8s.io/kind/pkg/cluster/constants (from $GOROOT)
	/Users/ashisha/go/src/sigs.k8s.io/kind/pkg/cluster/constants (from $GOPATH)
```

There are multiple references to `github.com/chuckha` which needs to be removed.

--- a/.gitignore
+++ b/.gitignore
@@ -1,0 +1,3 @@
+capdctl
+capd-manager
+kind-test

--- a/README.md
+++ b/README.md
@@ -6,10 +6,16 @@ A sample is built and hosted at `gcr.io/kubernetes1-226021/capd-manager:latest`
 
 ### Building the binaries
 
-Requires go with go modules.
+Requires go 1.12+ with go modules.
 
-* `go build ./cmd/...`
-* `go build ./cmd/`
+```
+# required if `cluster-api-provider-docker` was cloned into $GOPATH
+export GO111MODULE=on
+# build the binaries
+go build ./cmd/capdctl
+go build ./cmd/capd-manager
+go build ./cmd/kind-test
+```
 
 ### Building the image
 

